single-sign-on/configure/index.md

```diff
@@ -0,0 +1,158 @@
+
+1. Sign in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} as an administrator and navigate to **Organizations** and select the organization that you want to enable SSO on.
+2. Select **Settings** and select the **Security** tab.
+3. Select an authentication method for **Azure AD**.
+4. In the Identity Provider Set Up, copy the **Redirect URL / Reply URL**.
+
+    ![SSO Azure AD OIDC](/single-sign-on/images/sso-azure-oidc.png){:width="500px"}
+
+5. Sign in to your IdP to complete the IdP server configuration process. Refer to your IdP documentation for detailed instructions.
+
+    > **Note**
+    >
+    > The NameID is your email address and is set as the default.
+    > For example: yourname@mycompany.com.
+
+6. Complete the fields in the **Configuration Settings** section and click **Save**. If you want to change your IdP, you must delete your existing provider and configure SSO with your new IdP.
+
+    ![SSO Azure3](/single-sign-on/images/sso-azure3.png){:width="500px"}
+
+7. Proceed to **add your domain** before you test and enforce SSO.
+
+## Domain control
+
+Select **Add Domain** and specify the corporate domain you’d like to manage with SSO. Format your domains without protocol or www information, for example, yourcompany.com. Docker supports multiple domains that are part of your IdP. Make sure that your domain is reachable through email.
+
+> **Note**
+>
+> This should include all email domains and sub-domains users will use to access Docker.
+> Public domains such as gmail.com, outlook.com, etc aren't permitted.
+> Also, the email domain should be set as the primary email.
+
+![SSO Domain](/single-sign-on/images/sso-domain.png){:width="500px"}
+
+<hr>
+</div>
+</div>
+
+## Domain verification
+
+To verify ownership of a domain, add a TXT record to your Domain Name System (DNS) settings.
+
+1. Copy the provided TXT record value and navigate to your DNS host and locate the **Settings** page to add a new record.
+2. Select the option to add a new record and paste the TXT record value into the applicable field. For example, the **Value**, **Answer** or **Description** field.
+
+    Your DNS record may have the following fields:
+    * Record type: enter your 'TXT' record value
+    * Name/Host/Alias: leave the default (@ or blank)
+    * Time to live (TTL): enter **86400**
+
+3. After you have updated the fields, select **Save**.
+
+    > **Note**
+    >
+    > It can take up to 72 hours for DNS changes to take effect, depending on
+    > your DNS host. The Domains table will have an Unverified status during
+    > this time.
+
+4. In the Security section of your Docker organization, select **Verify** next to the domain you want to verify after 72 hours.
+
+    > **Note**
+    >
+    > Once you've verified your domain, you can move forward to test your
+    > configuration and enforce SSO, or you can configure your [System Cross-domain Identity Management (SCIM)](/docker-hub/scim.md).
```

Comment:
```suggestion
    > configuration and enforce SSO, or you can configure your [System Cross-domain Identity Management (SCIM)](../../docker-hub/scim.md).
```
I'm unable to test since this file moved, but I think that's correct.

Reply:
Thank you